GDJS/Runtime/pixi-renderers/layer-pixi-renderer.js

```diff
@@ -13,28 +13,49 @@
  * @param {gdjs.Layer} layer The layer
  * @param {gdjs.RuntimeScenePixiRenderer} runtimeSceneRenderer The scene renderer
  */
-gdjs.LayerPixiRenderer = function(layer, runtimeSceneRenderer) {
+gdjs.LayerPixiRenderer = function (layer, runtimeSceneRenderer) {
   this._pixiContainer = new PIXI.Container();
   /** @type Object.<string, gdjsPixiFiltersToolsFilter> */
   this._filters = {};
   this._layer = layer;
-  runtimeSceneRenderer.getPIXIContainer().addChild(this._pixiContainer);
 
+  /** @type {?PIXI.RenderTexture} */
+  this._renderTexture = null;
+
+  /** @type {?PIXI.Sprite} */
+  this._lightingSprite = null;
+
+  this._runtimeSceneRenderer = runtimeSceneRenderer;
+  this._pixiRenderer = runtimeSceneRenderer.getPIXIRenderer();
+  this._oldWidth = null;
+  this._oldHeight = null;
```

Comment:
Add a comment indicating that the width/height are tracked when a render texture is used.